src/font.h

```diff
@@ -16,9 +16,12 @@
 
 #define POS_CENTRED -0x8000
 
+#define DIALOGUE_NUM_NULL 0xFF
+
 enum TextFonts {
     FONT_COLOURFUL,
     FONT_SMALL,
+    FONT_UNK01 = 255,
```

Comment:
Maybe we can rename this FONT_UNKFF or FONT_UNK255 so it's clear that the value is not 01?